e2e/ui/api-client.js

```diff
@@ -25,6 +47,7 @@ export const client = async (
       "X-Nomad-Token": NOMAD_TOKEN,
       ...customHeaders,
     },
+    httpsAgent: httpsAgent,
```

Comment:
`nit`: this can just be `httpsAgent` instead of `httpsAgent: httpsAgent`

`question`: Do we always want this present? Or are we conditionally passing this as a configuration?

Reply:
> question: Do we always want this present? Or are we conditionally passing this as a configuration?

I think we do. We won't have configured the client certs for mTLS if the env vars aren't set appropriately. But we still can take advantage of the config if `NOMAD_ADDR` is https but mTLS isn't configured for some reason (a somewhat oddball local dev environment, I guess).